projects/plugins/jetpack/extensions/blocks/contact-form/util/with-style-variables.js

```diff
@@ -0,0 +1,10 @@
+import './form-styles.js';
+import { useCallback } from '@wordpress/element';
+
+export const withStyleVariables = WrappedComponent => props => {
+	const EDITOR_SELECTOR = '[data-type="jetpack/contact-form"]';
+	/* eslint-disable-next-line no-undef */
+	const generateVariables = useCallback( () => generateStyleVariables( EDITOR_SELECTOR ), [] );
```

Comment:
Nit: `window.generateStyleVariables` or `import { generateStyleVariables } from window;` might be a little more clear.